priv/perf/apps/load_test/lib/scenario/start_standard_exit.ex

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Scenario.StartStandardExit do
+  @moduledoc """
+  Starts a standard exit.
+
+  ## configuration values
+  - `exiter` the account that's starting the exit
+  - `utxo` the utxo to exit
+  """
+
+  use Chaperon.Scenario
+
+  alias Chaperon.Session
+  alias LoadTest.ChildChain.Exit
+
+  def run(session) do
```

Comment:
I am slightly concerned that we are over moduling on the scenario level where we wants really are some helper client functions to wrap a bit of complexity to make the call. Relying on session to pass around vars are not that great in my opinion as it is harder to follow at certain point, what var is in the session when we need to cross read multiple Scenario files in a single runner.

I would bias over to use sth like in the place that runs this scenario
```elixir
tx_hash = root_chain.start_standard_exit()
session = Session.assign(session, tx_hash: tx_hash)
```
if necessary. I think most of the cases we don't even need the part of assign it to session.

Same for the watcher info scenario

Reply:
I made it a scenario for 2 reasons:

1. Chaperon automatically adds metrics to scenarios, so you don't have to add them explicitly (as in your comment above)
2. Chaperon allows you compose scenarios, which can make it easier to create complex scenarios. e.g. to measure watcher status.get you just add `LoadTest.Scenario.WatcherStatus` to the list (see comment below).

But TBH I'm kind of in two minds about this. 
As you pointed out, the 'composability' is a bit weird - I think it can reduce code reuse in simple cases like `WatcherStatus`, but for more complex scenarios storing everything in the session map can be a bit confusing. 

The automatic metrics are nice to have, but can be added explicitly where necessary.

I'm fine with either way really...

Reply:
hmmm I am okay to continue like this. Will leave this to your decision.

Though I do believe client code would be more reusable and useful compare to this. But yeah, free metric is a selling point. 

Composability wise I am on the fence, lower level compatibility seems a bit more useful, at least I am not really buyed with the compatibility in this PR. But the previous example of composing scenario for create UTXO and spend UTXO was great. Probably to me it is more about the complexity your wrapping on 🤔 But again I am not strongly against this yet it is more of my intuition so far.